charts/umbrella/README.md

```diff
@@ -467,16 +469,34 @@ helm install -f values-adopter-portal.yaml umbrella . --namespace umbrella --cre
 
 #### Data exchange
 
-:warning: Please be aware of [Note for R24.05](#note-for-r2405)
-
 Involved components:
 
-EDC, MIW, DTR, Vault (data provider and consumer in tx-data-provider), CentralIdP.
+> DBRS-Server-Memory, Data Consumer, Data Provider, SSI Dim Wallet Stub.
+
+To start the Data Exchange, it is necessary to execute the [Data Exchange Subset](https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/umbrella-2.0.0/charts/umbrella/README.md#option-2-1)
+
+There are two ways to test and execute the Data Exchange tutorial:
+
+**1. Postman**
+
+You can import the Umbrella [Postman Collection](../../docs/api/postman/UmbrellaConnectorData-Exchange.postman_collection.json)
+
+**2. Curl**
 
-TBD.
+You can follow the [Curl Steps](../../docs/api/curl/UmbrellaConectorDataExchange.md)
 
 #### Get to know the Portal
 
+Involved components:
+
+> Portal, Centralidp, sharedidp, pgadmin4 
+
+Some components are disabled, as they are not necessary for the exclusive use of the portal. If you want to perform the onboarding of a participant, it would be necessary to enable the following components:
+
+> BPDM, SSI-Credential-Issuer, SSI-DIM-WALLET-STUB
```

Comment:
```suggestion
Some components are disabled, as they are not necessary for the exclusive use of the portal. If you want to perform the onboarding of a participant or register as well as discover a connector (EDC), it would be necessary to enable the following components:

- BPDM
- SSI DIM Wallet Stub
- BPN-DID-Resolution-Service (BDRS)
- SSI-Credential-Issuer
- Self Description Factory
- Discovery Finder
- BPN Discovery
```

Could you please add this note `# Set 'enabled' to true if onboarding is required` above the [sd-factory](https://github.com/mgarciaLKS/tractus-x-umbrella/blob/main/charts/umbrella/values-adopter-portal.yaml#L59) and add the bdrs as disabled as well to portal values file (with the same note)?